src/Compilers/VisualBasic/Portable/Scanner/KeywordTable.vb

```diff
@@ -297,7 +298,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax
 
             Dim kind As SyntaxKind
             If Not s_keywords.TryGetValue(tokenName, kind) Then
-                kind = SyntaxKind.IdentifierToken
+                ' s_keywords includes preprocessor keywords, but only one of
+                ' "r" and "reference".  GetProcessorKeywordKind supports both.
```

Comment:
At one point, the plan was to support both since the directive was viewed as a runtime version of the command-line switch, which supports both.  @tmat, are we still planning to support both?